### DISCUSSION TITLE
Why is fin called fin?

### DISCUSSION THREAD

**Discussion by seanmacgillivray** [docksal/docksal#1554] [Q&A] [answered]
We love Docksal here at my company, but we're all curious where the CLI got its name!

**Comment by shelane** [comment, marked as answer]
Docker + Dorsal = Docksal
The Dorsal is a fin!

**Reply by seanmacgillivray** [reply]
Awesome! Thanks for everything you do with Docksal, it is really game-changing for us here at Aptitude Digital.

**Reply by dpremo-ul** [reply]
What's Dorsal?

**Reply by seanmacgillivray** [reply]
> What's Dorsal?

The fin on top/back of a fish or whale is its dorsal fin. Like the shark fin that sticks up out of the water, for example.

**Reply by dpremo-ul** [reply]
I know that.  

The earlier response makes it sound like Dorsal is a development tool that is used in conjunction with Docker.  If it is, a link would be great.  If it isn't, then why combine the name Docker with Dorsal?

**Reply by seanmacgillivray** [reply]
> I know that.
> 
> The earlier response makes it sound like Dorsal is a development tool that is used in conjunction with Docker. If it is, a link would be great. If it isn't, then why combine the name Docker with Dorsal?

I think it's in keeping with the fact that Docker uses a whale for its logo? Whales do have dorsal fins, although many are vestigial like the funny rear-flipper bones Blue whales have.
